### PR TITLE
Add a new theme 'so-clean', mock `currentUserID` with `req.query.currentUser`

### PR DESCRIPTION
**Pull request by moesoha**
The design of theme _so-clean_ comes from [Hashable/so-clean](https://github.com/Hashable/so-clean).

### CODE REVIEW

File: src/templates/so-clean/components/pagination.vue [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<template>
+	<nav class="pagination" role="navigation">
+		<a class="newer-posts" @click="callback(page - 1)" v-if="page>1"><span aria-hidden="true">&larr;</span> 新点儿的</a>
```
Comment (laosb) [inline src/templates/so-clean/components/pagination.vue]: Better lint code in vue expressions too, by yourself. Add spaces before and after operators.

File: src/templates/so-clean/styles/screen.css [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+	margin: 0 0 1em;
+}
+
+img {
+	max-width: 100%;
+}
+
+iframe {
+	max-width: 100%;
+}
```
Comment (laosb) [inline src/templates/so-clean/styles/screen.css]: Always put a new line at the end of a source code file.

**Comment by moesoha** [comment]
Changes committed. @laosb

**Comment by laosb** [comment]
LGTM

**Pull request merged**